recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "8.0.1" %}
+{% set version = "8.1.0" %}
 
 package:
   name: proj
   version: {{ version }}
 
 source:
   url: http://download.osgeo.org/proj/proj-{{ version }}.tar.gz
-  sha256: e0463a8068898785ca75dd49a261d3d28b07d0a88f3b657e8e0089e16a0375fa
+  sha256: 22c5cdc5aa0832077b16c95ebeec748a0942811c1c3438c33d43c8d2ead59f48
 
 build:
   number: 0
```

Comment:
The automerge PR created a build 0 for this version already. Let's bump here to 1/